src/aind_metadata_service/response_handler.py

```diff
@@ -119,3 +121,9 @@ def map_to_json_response(self) -> JSONResponse:
         else:
             response = self._map_data_response()
         return response
+
+    def map_to_pickled_response(self) -> Response:
+        """Map a ModelResponse to a pickled response."""
+        return Response(
```

Comment:
We should also handle mapping the status code here. We may also want to consider just returning `content=pickle.dumps(self.aind_models)`.